model/cloud.go

```diff
@@ -105,6 +105,10 @@ type ValidateBusinessEmailRequest struct {
 	Email string `json:"email"`
 }
 
+type ValidateBusinessEmailResponse struct {
+	IsValid string `json:"is_valid"`
```

Comment:
nit: I don't see a reason to not use a `bool` here, we always want to return `true` or `false`.

Overall I believe the more accurate the type, the better quality of life for maintenance etc.

Reply:
ditto it would be better to represent as a bool. It would simplify the code in the frontend as well.